lib/features/thread/domain/constants/thread_constants.dart

```diff
@@ -17,6 +17,7 @@ class ThreadConstants {
     EmailProperty.sentAt,
     EmailProperty.preview,
     EmailProperty.hasAttachment,
+    EmailProperty.headers,
```

Comment:
Please do not request the full headers. Only the one you need.

Reply:
@sakriwassim please notice to it, and show the demo with postman first